spec/lib/areacodes_spec.rb

```diff
@@ -52,6 +52,14 @@
     end
   end
 
+  describe '.area_codes_by_state_code' do
+    it 'returns area_codes' do
+      codes = described_class.area_codes_by_state_code(1)
+      expected = [205, 251, 256, 334, 938].sort
+      expect(codes.sort).to eql(expected)
```

Comment:
Tiny thing but can use `match_array` (which doesn't care about order) https://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers:match_array to avoid having to sort